AddisAbabaNALE/NLA23.xml

```diff
@@ -53,7 +53,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                <msContents>
                   <summary>EMML: Enoch, fol. 3r-56v; 1 Maccabees, fol. 57r-86r; 2 Maccabees, fol. 86r-103r; 3 Maccabees, fol. 103r-112r. EMIP: Enoch, መጽሐፈ፡ ሄኖክ፡ (p. 5); 1 Maccabees (p. 113); 2 Maccabbees (p. 171); 3 Maccabbees (p. 205).    
                   </summary>
-                  
+                  <msItem xml:id="ms_i1">
+                     <locus from="3r" to="56v"/>
```

Comment:
as already mentioned I do not think this is helpful for the project.

either not point or point to all.

Reply:
Ok, I will encode other texts as well.